management/templates/login.html

```diff
@@ -128,6 +132,9 @@ <h1 style="margin: 1em; text-align: center">{{hostname}}</h1>
     localStorage.removeItem("miab-cp-credentials");
   if (typeof sessionStorage != 'undefined')
     sessionStorage.removeItem("miab-cp-credentials");
+
+  $("#guestNav").addClass("hidden");
```

Comment:
These seem to be in the wrong order, therefore the loggedInNav is not hiding when you are logging out